include/mxnet/storage.h

```diff
@@ -32,6 +32,8 @@
 
 namespace mxnet {
 
+using handle_id_t = unsigned long long;
+using timestamp_t = unsigned long long;
```

Comment:
This two definitions should no longer be needed since they are included in swap.h